scripts/import_osm.py

```diff
@@ -0,0 +1,111 @@
+            raise AssertionError("You must specify the city name.")
+        # BBOX (minx, miny, maxx, maxy) takes precedence over city
+        arg_bbox = args["bbox"]
+        if arg_bbox:
+            minx, miny, maxx, maxy = map(float, arg_bbox.split(", "))
+            city_name = ""
+        else:
+            # support old city parameter too, if we want to let osmnx do geocoding
+            if slug == "COP":
+                city_name = "Copenhagen, Denmark"
+                minx, miny, maxx, maxy = 12.42000, 55.61000, 12.65000, 55.78000
+            elif slug == "HEL":
+                city_name = "Helsinki, Finland"
+                minx, miny, maxx, maxy = 24.82345, 60.14084, 25.06404, 60.29496
+            elif slug == "WAR":
+                city_name = "Warsaw, Poland"
+                minx, miny, maxx, maxy = 20.79057, 52.09901, 21.31300, 52.38502
+            else:
+                raise AssertionError("You must specify a bounding box or a known city.")
+
+        return {"slug": slug, "bbox": (minx, miny, maxx, maxy), "city_name": city_name}
+
+    def _create_engine(self) -> Engine:
+        """Sets up and returns a DB engine."""
+        sql_url = get_connection_url(dbname="geoviz")
+        engine = create_engine(sql_url)
+        return engine.execution_options(
+            schema_translate_map={"schema": self.slug}
+        )
+
+    def run(self):
+        self._initialise_db()
+        print("Fetching OSM data from overpass API...")
+        pois = self._get_amenities()
+        pois = pois.to_crs(epsg=3035)
+        pois.geometry = pois.centroid
+        pois = pois.to_crs(epsg=4326)
+
+        pois = pois.reset_index()
+        tag_columns = list(pois.columns)
+        tag_columns.remove("osmid")
+        tag_columns.remove("geometry")
+
+        pois = pois.rename(columns={"osmid": "node_id"})
+        pois = pois.rename(columns={"geometry": "geom"})
+        pois = pois.set_index("node_id")
+        pois["geom"] = pois["geom"].apply(lambda geom: WKTElement(geom.wkt, srid=4326))
+
+        pois["tags"] = pois[tag_columns].apply(lambda x: x.to_json(), axis=1)
+        pois = pois.drop(tag_columns, axis=1)
+
+        pois.to_sql(name=OSMPoint.__tablename__, con=self._engine, schema=self.slug, if_exists="append",
+                    dtype={"geom": Geometry(geometry_type="POINT", srid=4326)})
+
+    def _initialise_db(self) -> None:
+        """Initialises OSM points table and returns a new DB Session."""
+        with self._engine.connect() as con:
+            con.execute("CREATE EXTENSION postgis")
+        self._engine.execute(CreateSchema(self.slug))
+
+        OSMPoint.__table__.drop(self._engine, checkfirst=True)
+        OSMPoint.__table__.create(self._engine)
+
+    def _get_amenities(self):
+        try:
+            (minx, miny, maxx, maxy) = self.bbox
+            return ox.geometries.geometries_from_bbox(maxy, miny, maxx, minx, tags=tags_to_filter)
+        except (TypeError, ValueError):
```

Comment:
Probably we don't need this? Seems the importer will not run without providing a bounding box anyway (apart from the three old slugs HEL, COP, WAR that we should deprecate)?

Or alternatively, if we want to keep the ox geocoding from city name, AssertionError in _parse_args should only check that *either* bbox *or* city is set. But simpler would be to just require bbox.